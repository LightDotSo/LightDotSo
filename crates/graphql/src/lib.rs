pub mod constants;
pub mod polling;
pub mod utils;
pub mod wallets;
