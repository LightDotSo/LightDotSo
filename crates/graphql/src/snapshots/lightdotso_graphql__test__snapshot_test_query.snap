---
source: crates/graphql/src/lib.rs
expression: query.query
---
query MyQuery($n: Int!) {
  lightWallets(first: $n) {
    id
  }
}


